mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketPublisher.java

```diff
@@ -56,4 +58,8 @@ public void close() {
             producer.close();
         }
     }
+
```

Comment:
This is less random than before, but it's still random. 

I was initially thinking of suggesting that it might be useful tot he customer to be able to explicitly name their consumers. For reasons such as managing them, recognising their purpose and, maybe most importantly, so they/we can trace them in server logs.

But I think there's a case for not having any subscriber names generated by the library at all - if this subscriber crashes and is restarted, it will generate a new random name and a rogue/zombie subscriber (the old one) gets left behind.

I think for the reasons we've seen with stale/discarded subscribers, we should expect them to always have a explicit name.

Reply:
Yes, it is an explicit name for Subscriber and for Producers, it will be a hardcoded one.

Reply:
Sorry - I think I might have miss-read the PR changes (seeing randomAphabetic, but it's red/deleted - not added and not in the code). I think I was half asleep when I reviewed. Subscriber name added in config class @builder class.

Sorry for the panic